setup.py

```diff
@@ -0,0 +1,29 @@
+# noqa: D100
+import os
+from setuptools import setup, find_packages
+
+with open(os.path.join(os.path.abspath(os.path.dirname(__file__)), "version.txt")) as f:
+    version = f.read().rstrip()
+
+setup(
+    name="treescript",
+    version=version,
+    description="Tree Modifying Script",
+    author="Mozilla Release Engineering",
+    author_email="release+python@mozilla.com",
+    url="https://github.com/mozilla-releng/treescript",
+    packages=find_packages(),
+    package_data={"signingscript": ["data/*"]},
```

Comment:
When running `python setup.py sdist`, it looks like we're missing robustcheckout. We either need to include that here or in `MANIFEST.in`. Also, signingscript.

Reply:
Good catch, thanks.